docs/alpine.md

```diff
@@ -0,0 +1,212 @@
+# Updating Alpine Base
```

Comment:
I'm in two minds about factoring this out, but if we do the file name needs to be more descriptove. like `alpine-base-update.md`

Reply:
I don't mind changing the name.

> I'm in two minds about factoring this out, 

I was, as well, until I looked back on what I went through updating alpine. This was 100% _not_ a release, yet had to go through all of the process, and while IW as at it, figure out where to start, which parts of the doc made sense just for a release vs for updating alpine, etc. It really needed a standalone "here is how to update alpine and its dependencies" doc.

Reply:
File renamed.